### PR TITLE
Set up PHPStan

### PR DESCRIPTION
**Pull request by szepeviktor**
@aristath

### CODE REVIEW

File: includes/wptt-webfont-loader.php [hunk 1]
```diff
@@ -263,15 +263,15 @@ public function get_remote_url_contents() {
...
 
 			// Get the CSS from our response.
 			$contents = wp_remote_retrieve_body( $response );
 
 			// Early exit if there was an error.
-			if ( is_wp_error( $contents ) ) {
-				return;
+			if ( '' === $contents ) {
+				return '';
 			}
```
Comment (szepeviktor) [inline includes/wptt-webfont-loader.php]: This should be deleted, `wp_remote_retrieve_body` returns the body or an empty string.

File: includes/RequireGutenberg.php [hunk 2]
```diff
@@ -68,7 +68,7 @@ public function admin_notice() {
 			return;
 		}
 		?>
-		<div class="notice notice-warning require-gutenberg-notice-wrapper notice-alt<?php echo ( $active_step ) ? ' active-step-' . esc_attr( $active_step ) : ''; ?>">
+		<div class="notice notice-warning require-gutenberg-notice-wrapper notice-alt<?php echo ' active-step-' . esc_attr( $active_step ); ?>">
```
Comment (aristath) [inline includes/RequireGutenberg.php:71]: Why this change? :thinking:

Comment (szepeviktor) [reply includes/RequireGutenberg.php:71]: `$active_step` is checked above thus it is not possible to be falsy

Comment (aristath) [reply includes/RequireGutenberg.php:71]: Good point, I missed it. Thanks!

**Review by aristath — APPROVED** [review]

File: includes/RequireGutenberg.php (hunk 2 above)
Comment (szepeviktor) [reply includes/RequireGutenberg.php:71]: @phpstan does not miss what we humans overlook.

**Comment by szepeviktor** [comment]
You're welcome.
